setup.py

```diff
@@ -148,9 +148,10 @@
 
 def check_linker_need_libatomic():
     """Test if linker on system needs libatomic."""
+    cc = os.environ.get('CC', 'cc')
```

Comment:
Nice catch! But I think we probably want to be using [this interface](https://docs.python.org/3/distutils/apiref.html#module-distutils.ccompiler).